From 43c31e3c2be8cf21be67172301b70527a9ea6b34 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
 <frederic.epitre@orange.fr>
Date: Fri, 19 Oct 2018 08:02:12 +0200
Subject: [PATCH] anaconda: drop useless on Qubes dependencies on network
 filesystems

Installing Qubes on network drive is not supported, so drop those dependencies.
---
 pyanaconda/kickstart.py | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/pyanaconda/kickstart.py b/pyanaconda/kickstart.py
index 7d67bf1a6..1121e0928 100644
--- a/pyanaconda/kickstart.py
+++ b/pyanaconda/kickstart.py
@@ -2155,12 +2155,6 @@ def parseKickstart(f):
 
     # We need this so all the /dev/disk/* stuff is set up before parsing.
     udev.trigger(subsystem="block", action="change")
-    # So that drives onlined by these can be used in the ks file
-    blivet.iscsi.iscsi.startup()
-    blivet.fcoe.fcoe.startup()
-    blivet.zfcp.zfcp.startup()
-    # Note we do NOT call dasd.startup() here, that does not online drives, but
-    # only checks if they need formatting, which requires zerombr to be known
 
     try:
         ksparser.readKickstart(f)
-- 
2.17.1

